diesel_cli/README.md

```diff
@@ -9,7 +9,7 @@ Getting Started
 
 ```shell
 cargo install diesel_cli
-diesel setup --database-url='postgres://localhost/my_db'
+diesel setup
```

Comment:
This line shouldn't change, unless we indicate that there's a `.env` file expected.